src/components/tools/table-tool/table-tool.scss

```diff
@@ -53,9 +53,8 @@ $controls-hover-background: #c0dfe7;
       }
 
       .link-geometry-button {
-        position: absolute;
-        right: 3.5px;
-        top: 3px;
+        margin-top: -23px;
+        margin-left: calc(100% - #{$index-column-width} + #{$table-padding});
```

Comment:
```suggestion
        position: absolute;
        right: 3.5px;
        top: 3px;
```
The code as previously written was clearly designed to absolutely position the element relative to the top-right corner of the cell, which is what we actually want. The reason that's no longer working is presumably because something about the parent element that it is being absolutely positioned relative to has changed. I believe that adding `position: relative` to the `.editable-header-cell` element will fix the problem in a more intuitive and maintainable manner. One way to see this is that your version doesn't handle the case when the title has wrapped onto multiple lines properly.

Reply:
Okay, so it looks like `editable-header-cell` is this orange block
<img width="619" alt="image" src="https://user-images.githubusercontent.com/521934/192841898-65dede7d-5655-404f-b9e6-1ea4920c63bb.png">
<img width="349" alt="image" src="https://user-images.githubusercontent.com/521934/192841934-44106d55-9904-40b1-bdd5-0e4e1a2346ca.png">

So instead I added` position: relative` to `.table-title` (seen in blue) 
 
<img width="406" alt="image" src="https://user-images.githubusercontent.com/521934/192842673-b25691c5-09c1-425e-94c0-9371aeb7879f.png">

<img width="1122" alt="image" src="https://user-images.githubusercontent.com/521934/192842766-7558d351-e28b-4e66-8908-f5d13b019af3.png">
Added lines 37,38, 60, 61, 63.

Overall I tested it by adding multiple columns, and a title that has wrapped onto multiple lines and the icon stays fixed. 
I'll clean up and resubmit.